cpu-on-dns-pods-high-variables.tf

```diff
@@ -0,0 +1,66 @@
+variable "cpu_on_dns_pods_high_enabled" {
+  type    = bool
+  default = true
+}
+
+variable "cpu_on_dns_pods_high_warning" {
+  type    = number
+  default = 70
+}
+
+variable "cpu_on_dns_pods_high_critical" {
+  type    = number
+  default = 85
+}
+
+variable "cpu_on_dns_pods_high_evaluation_period" {
+  type    = string
+  default = "last_5m"
+}
+
+variable "cpu_on_dns_pods_high_severity" {
+  type    = string
+  default = "major"
+}
+
+variable "cpu_on_dns_pods_high_note" {
+  type    = string
+  default = ""
+}
+
+variable "cpu_on_dns_pods_high_docs" {
+  type    = string
+  default = ""
+}
+
+variable "cpu_on_dns_pods_high_filter_override" {
+  type    = string
+  default = "kube_service:kube-dns OR short_image:coredns OR short_image:ucp-coredns OR short_image:ucp-kube-dns"
```

Comment:
Let's leave this as the default but make sure that you also set this override in the k8s module configuration. Because we also should specify the cluster name.

Reply:
Bedoel je hier? https://cde.toyota-europe.com/stash/projects/DAT/repos/datadog-terraform/browse/services/kubernetes.tf#12

Ik zou even niet weten waar het nog meer kan dan?